.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
 before_install:
   - npm install -g npm@^3 --cache-min 999999999
   - npm install -g coveralls --cache-min 999999999
-  - time npm i -g yarn --cache-min 999999999
```

Comment:
Why did you remove this in the PR?

Reply:
We don't use yarn at all. Why install it alongside npm?

Reply:
Good point! I think that the travis.yml could be cleaned up a bit as well, lot of unused stuff in there.

Reply:
good for a follow-up PR

Reply:
BTW - I use yarn, I just didn't want to unnecessarily change here.  But I don't mind switching if others want to.